ExampleMod/ExampleMod.ModCalls.cs

```diff
@@ -11,17 +12,19 @@ namespace ExampleMod
 	// This class extends from the Mod class as seen in ExampleMod.cs. Make sure to extend from the mod class, ": Mod", in your own code if using this file as a template for you mods Mod class.
 	partial class ExampleMod
 	{
+		public static LocalizedText[] ErrorText { get; private set; }
+
 		// The following code allows other mods to "call" Example Mod data.
 		// This allows mod developers to access Example Mod's data without having to set it a reference.
 		// Mod calls are not exposed by default, so it will be up to you to publish appropriate calls for your mod, and what values they return.
 		public override object Call(params object[] args) {
 			// Make sure the call doesn't include anything that could potentially cause exceptions.
 			if (args is null) {
-				throw new ArgumentNullException(nameof(args), "Arguments cannot be null!");
+				throw new ArgumentNullException(ErrorText[0].Format(nameof(args)));
```

Comment:
Honestly, `ArgumentNullException(nameof(args))` is plenty here, the text was unnecessary to start with.